### PR TITLE
Set IMAGE_PREFIX_ALT for explicitly for tests only, only release versioned virtctl binaries

### LINKED ISSUE #2352: Release includes versioned and unverisoned binaries
/kind bug

**What happened**:
The v0.18.0 release contains ersioned and unversioned binaries as seen in the screenshow (virtctl-v0.18.0-darwin-amd64 and virtctl-dawrin, same for windows).

**What you expected to happen**:
Only versioned binaries

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### LINKED ISSUE #2778: Images with `kv-` image name prefix were pushed
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
Images with `kv-` image name prefix were released for v0.20.8 (see #2776)

**What you expected to happen**:
No new images with the prefix

**Anything else we need to know?**:
I deleted the images already
Probably it is enough to set IMAGE_PREFIX_ALT="" in the build step which pushes images

### PR DESCRIPTION
**Pull request by slintes**
**What this PR does / why we need it**:
Set IMAGE_PREFIX_ALT for explicitly for tests only, only release versioned virtctl binaries

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2352 
Fixes #2778 

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: tests/operator_test.go [hunk 1]
```diff
@@ -940,7 +940,7 @@ spec:
 		})
 
 		// this test ensures that we can deal with image prefixes in case they are not used for tests already
-		It("should be able to create kubevirt install with image prefix", func() {
+		FIt("should be able to create kubevirt install with image prefix", func() {
```
Comment (stu-gott) [inline tests/operator_test.go]: Is this a WIP PR?

Comment (slintes) [reply tests/operator_test.go]: crap, no, thx :)

**Pull request merged**